raphtory/src/search/mod.rs

```diff
@@ -76,12 +82,11 @@ pub(in crate::search) mod fields {
     pub const VERTEX_ID_REV: &str = "node_id_rev";
     pub const NAME: &str = "name";
     pub const NODE_TYPE: &str = "node_type";
-    // edges
-    // pub const SRC_ID: &str = "src_id";
+    pub const EDGE_ID: &str = "edge_id";
     pub const SOURCE: &str = "from";
-    // pub const DEST_ID: &str = "dest_id";
     pub const DESTINATION: &str = "to";
-    pub const EDGE_ID: &str = "edge_id";
+    pub const CONSTANT_PROPERTIES: &str = "constant_properties";
+    pub const TEMPORAL_PROPERTIES: &str = "temporal_properties";
```

Comment:
would it be better name these `cprop` and `tprop`? @miratepuffin

Reply:
these are a lot of characters to add to the query in addition to the property name